.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -0,0 +1,47 @@
+name: Bug Report
+description: Something is not working correctly or is not working at all!
+title: "Bug: <Insert Issue Title Here>"
+labels: ["bug", "programming"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Example:** Bug: derive_vars_dtm() is returning NA values when using preserve argument
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info or to let you know when the bug has been fixed?
+      placeholder: email@example.com
```

Comment:
Do we really need this? You can only submit an issue with a GitHub account so we already have a line of communication.